e2etests/common.go

```diff
@@ -0,0 +1,12 @@
+package e2etests
+
+import (
+	"fmt"
+
+	"github.com/onsi/ginkgo"
+)
+
+//nolint:errcheck
```

Comment:
Is it possible to resolve the linter issue? We should avoid nolint until necessary.

Reply:
Yes, I can remove this.